src/app/register/components/form-visibility/form-visibility.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, forwardRef } from '@angular/core'
+import { BaseForm } from '../BaseForm'
+import {
+  FormGroup,
+  FormControl,
+  NG_VALUE_ACCESSOR,
+  NG_VALIDATORS,
+  Validators,
+  NG_ASYNC_VALIDATORS,
+} from '@angular/forms'
+import { RegisterService } from 'src/app/core/register/register.service'
+
+@Component({
+  selector: 'app-form-visibility',
+  templateUrl: './form-visibility.component.html',
+  styleUrls: ['./form-visibility.component.scss'],
+  preserveWhitespaces: true,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => FormVisibilityComponent),
+      multi: true,
+    },
+    {
+      provide: NG_ASYNC_VALIDATORS,
+      useExisting: forwardRef(() => FormVisibilityComponent),
+      multi: true,
+    },
+  ],
+})
+export class FormVisibilityComponent extends BaseForm implements OnInit {
+  visibilityOptions = ['PUBLIC', 'LIMITED', 'PRIVATE']
```

Comment:
@leomendoza123 I think we need to add these on the constants files since we will reuse it later

Reply:
true!